src/swarm/neo/client/mixins/AllNodesRequestCore.d

```diff
@@ -140,8 +141,14 @@ public struct AllNodesRequest ( Request, Connector, Disconnected, Initialiser,
 
                 this.handler();
             }
-            // Only retry in the case of a connection error. Other errors
-            // indicate internal problems and should not be retried.
+            // Only retry in the case of a connection I/O error or error event
+            // in epoll. Other errors indicate internal problems and should not
+            // be retried.
+            catch ( IOError e )
```

Comment:
Still, there's a question about `IOWarning`. `IOWarning` should indicate remote party hangup. I think in this case we should reconnect as well.